flexget/plugins/list/series_list.py

```diff
@@ -0,0 +1,490 @@
+    series_id = Column(Integer, ForeignKey(SeriesListSeries.id))
+
+
+class SeriesListIDRegexp(Base):
+    __tablename__ = 'series_list_id_regexps'
+
+    id = Column(Integer, primary_key=True)
+    regexp = Column(Unicode)
+    added = Column(DateTime, default=datetime.now)
+    series_id = Column(Integer, ForeignKey(SeriesListSeries.id))
+
+
+class SeriesListQuality(Base):
+    __tablename__ = 'series_list_quality'
+
+    id = Column(Integer, primary_key=True)
+    quality = Column(Unicode)
+    added = Column(DateTime, default=datetime.now)
+    series_id = Column(Integer, ForeignKey(SeriesListSeries.id))
+
+
+class SeriesListQualities(Base):
+    __tablename__ = 'series_list_qualities'
+
+    id = Column(Integer, primary_key=True)
+    quality = Column(Unicode)
+    added = Column(DateTime, default=datetime.now)
+    series_id = Column(Integer, ForeignKey(SeriesListSeries.id))
+
+
+class SeriesListFromGroup(Base):
+    __tablename__ = 'series_list_from_groups'
+
+    id = Column(Integer, primary_key=True)
+    group_name = Column(Unicode)
+    added = Column(DateTime, default=datetime.now)
+    series_id = Column(Integer, ForeignKey(SeriesListSeries.id))
+
+
+class SeriesListSpecialID(Base):
+    __tablename__ = 'series_list_special_ids'
+
+    id = Column(Integer, primary_key=True)
+    id_name = Column(Unicode)
+    added = Column(DateTime, default=datetime.now)
+    series_id = Column(Integer, ForeignKey(SeriesListSeries.id))
+
+
+def get_db_series(entry):
+    title = entry.get('series_name') or entry['title']
+    db_series = SeriesListSeries(title)
+    # Setting series attributes only for received data
+    if entry.get('alternate_name'):
+        db_series.alternate_name = [SeriesListAlternateName(name=name) for name in entry.get('alternate_name')]
+    if entry.get('name_regexp'):
+        db_series.name_regexp = [SeriesListNameRegexp(regexp=regexp) for regexp in entry.get('name_regexp')]
+    if entry.get('ep_regexp'):
+        db_series.ep_regexp = [SeriesListEpRegexp(regexp=regexp) for regexp in entry.get('ep_regexp')]
+    if entry.get('date_regexp'):
+        db_series.date_regexp = [SeriesListDateRegexp(regexp=regexp) for regexp in entry.get('date_regexp')]
+    if entry.get('sequence_regexp'):
+        db_series.sequence_regexp = [SeriesListSequenceRegexp(regexp=regexp) for regexp in entry.get('sequence_regexp')]
+    if entry.get('id_regexp'):
+        db_series.id_regexp = [SeriesListIDRegexp(regexp=regexp) for regexp in entry.get('id_regexp')]
+    if entry.get('date_dayfirst'):
+        db_series.date_dayfirst = entry.get('date_dayfirst')
+    if entry.get('date_dayfirst'):
+        db_series.date_dayfirst = entry.get('date_dayfirst')
+    if entry.get('quality'):
+        db_series.quality = SeriesListQuality(quality=entry.get('quality'))
+    if entry.get('qualities'):
+        db_series.qualities = [SeriesListQualities(quality=quality) for quality in entry.get('qualities')]
+    if entry.get('timeframe'):
```

Comment:
Yeah, but looks like most of them are just moving value from object to another so those could be done with that approach. Rest can be still done separately.